### PR TITLE
59 use case get latest stories

### PR DESCRIPTION
**Pull request by sarkar50-lab**

### CODE REVIEW

**Review by bac423 — DISMISSED** [review]
Alright

**Review by cotopaxi64 — DISMISSED** [review]
Looks good to me.

File: src/main/java/usecases/get_latest_stories/GlsInputData.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+/**
+ * Input data class of Get Latest Stories use-case
+ */
+public class GlsInputData {
+
+    private final Integer numToGet;
+
+    /**
+     * Constructor for GlsInputData
+     * @param numToGet number of stories to get
```
Comment (AlekseyPanas) [inline src/main/java/usecases/get_latest_stories/GlsInputData.java:12]: Mention that this parameter can be provided as null to retrieve all stories

File: src/main/java/usecases/get_latest_stories/GlsOutputData.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+     * Constructor for GlsOutputData
+     * @param stories StoryData
+     */
+    public GlsOutputData (StoryData[] stories) {
+        this.stories = stories;
+    }
+
+    /**
+     * Getter for GlsOutputData
+     * @return  data.numToGet StoryData retrieved from repo
```
Comment (AlekseyPanas) [inline src/main/java/usecases/get_latest_stories/GlsOutputData.java:22]: also mention that they are sorted from latest date to earliest public date

**Review by AlekseyPanas — CHANGES_REQUESTED** [review]
Tests are perfectly documented and coverage is wonderful. The implementation of the use case is also mostly good but could use clarifying documentation in many areas. Please address my documentation remarks

**Review by joel-mat-03 — APPROVED** [review]
Code looks good and all conventions are followed

**Review by bac423 — APPROVED** [review]
[Quality Manager] Approved. Great job

**Pull request merged**